modules/UI/authentication/AuthHandler.js

```diff
@@ -43,7 +45,7 @@ function doExternalAuth(room, lockPassword) {
             getUrl = room.getExternalAuthUrl(true);
         }
         getUrl.then(url => {
-            externalAuthWindow = LoginDialog.showExternalAuthDialog(
+            externalAuthWindow = openAuthDialog(
```

Comment:
Will this really work? Currently that will directly open the popup to the given URL, but with your change the WaitForOwnerDialog will be open instead.

Reply:
Reverted the change.